app/src/main/java/woowacourse/paint/model/Painting.kt

```diff
@@ -0,0 +1,25 @@
+package woowacourse.paint.model
+
+import android.graphics.Canvas
+import android.graphics.Paint
+import androidx.annotation.ColorInt
+
+interface Painting {
+    val paint: Paint
+    fun movePath(x: Float, y: Float): Painting
+    fun initPath(prevX: Float, prevY: Float, x: Float, y: Float)
+    fun draw(canvas: Canvas)
+    fun setStroke(value: Float): Painting
+    fun setColor(@ColorInt color: Int): Painting
+
+    fun getNewPainting(): Painting
+
+    fun setPaintBrush(brushTool: BrushTool): Painting {
```

Comment:
```suggestion
    fun from(brushTool: BrushTool): Painting {
```
해당 객체를 가져오는 것이기 때문에 set보다는 from을 사용하는 것이 어떨까요??